scripts/deploy

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+build_chart() {
+  if [[ -z "${DOCKERHUB_USER}" ]]; then
+    echo "Please set env DOCKERHUB_USER to your dockerhub username"
+    exit 0
+  fi
+  #In the case of trying to build on non amd64 hosts, docker-buildx allows you to
+  #Cross compile for different architectures.
+  if [ "$ARCH" != "amd64" ]; then
+    echo "Building using docker-buildx"
+    docker buildx use default
+    make build
+    docker buildx use m1builder  
+    docker buildx build --platform linux/amd64 -f ../package/Dockerfile . -t $DOCKERHUB_USER/backup-restore-operator:$TAG --push
+
+  else
+    make build
+    docker build -f ../package/Dockerfile . -t $DOCKERHUB_USER/backup-restore-operator:$TAG --push
```

Comment:
This is what `scripts/package` does, we can't duplicate anything in the process as updating one would miss the other.

Reply:
So then should this alternative build method be moved to `scripts/package`? How do we package this docker image locally and then deploy it? Or was that one of the original questions?